hscontrol/handlers.go

```diff
@@ -134,38 +135,37 @@ func (h *Headscale) HealthHandler(
 	respond(nil)
 }
 
-type registerWebAPITemplateConfig struct {
-	Key string
+var codeStyleRegisterWebAPI = styles.Props{
+	styles.Display:         "block",
+	styles.Padding:         "20px",
+	styles.Border:          "1px solid #bbb",
+	styles.BackgroundColor: "#eee",
 }
 
-var registerWebAPITemplate = template.Must(
-	template.New("registerweb").Parse(`
-<html>
-	<head>
-		<title>Registration - Headscale</title>
-		<meta name=viewport content="width=device-width, initial-scale=1">
-		<style>
-			body {
-				font-family: sans;
-			}
-			code {
-				display: block;
-				padding: 20px;
-				border: 1px solid #bbb;
-				background-color: #eee;
-			}
-		</style>
-	</head>
-	<body>
-		<h1>headscale</h1>
-		<h2>Machine registration</h2>
-		<p>
-			Run the command below in the headscale server to add this machine to your network:
-		</p>
-		<code>headscale nodes register --user USERNAME --key {{.Key}}</code>
-	</body>
-</html>
-`))
+func registerWebHTML(key string) *elem.Element {
```

Comment:
Lets make a templates package, `templates`, rename it to something like `RegisterWeb` and then we can make more of the generic HTML reusable.